discovery-provider/src/models/models.py

```diff
@@ -581,6 +581,21 @@ def __repr__(self):
 play_item_id={self.play_item_id},\
 count={self.count})>"
 
+
+class HourlyPlayCounts(Base):
+    __tablename__ = "hourly_play_counts"
+
+    hourly_timestamp = Column(DateTime, primary_key=True, nullable=False)
+    play_count = Column(Integer, nullable=False, index=False)
+
+    Index("hourly_timestamp_idx", "hourly_timestamp", unique=True)
```

Comment:
since this is a primary key, do we need unique index here?

Reply:
yeah this is actually not necessary. primary_key takes care of it already. verified with `HourlyPlayCounts.metadata.tables['hourly_play_counts'].primary_key` in debugger. i plan on cleaning up these schemas later to make sure they match the tables.